### ISSUE TITLE
Clone an issue to another repo

### ISSUE THREAD

**Issue by dsollfrank** [codetree/feedback#319]
Moving issues from one repo to another is possible already. It would be very helpful to have the possibility to clone an issue to another repo or even better to multiple repos.

**Comment by ac21** [comment]
@dsollfrank - would you invision the cloned issues be synced (changes to one are replicated in another), linked in some way (similiar to needs/needed_by) or is this just a duplication?

also, do you mind expanding on your use case behind the need for cloning?

**Comment by dsollfrank** [comment]
@ac21 Thanks for the questions.

No sync, just duplication. A link of some sort would be nice of course.

We recently started with a new codetree project that links 13 repos. These are repos from several "departments". 
1) We create epics (thanks for that feature!) for overlapping tasks. 
2) For each involved department we create an issue that holds the basic task description and link to the epic and would be the same for each department.
3) Each department will have to detail the basic description of course

For step 2 a "cloning" functionality would be very helpful.

Basically we envisioned it to do more or less the same as the currently available "move to another repo" function without closing the issue in the original repo. Plus it would be great if multiple repos could be selected.

Hope that clarifies the request. Looking forward to feedback.

**Comment by dsollfrank** [comment]
@ac21 Will this issue be resolved any time soon?
